### PR TITLE
dashboard footer bug and footer nav mobile bug

### PR DESCRIPTION
**Pull request by RB-Dev94**
footer-nav on main site mobile fix and footer_bottom mobile fix

### CODE REVIEW

File: ui/src/layouts/Footer/index.tsx [hunk 1]
```diff
@@ -106,6 +115,17 @@ const FooterWrapper = styled.footer`
     }
   }
   .footer__bottom {
+    .flex {
```
Comment (EricSSartorius) [inline ui/src/layouts/Footer/index.tsx]: let's try to avoid putting styles on reserved utility classes, it might make more sense to give this a different class name and put the styles on there instead

**Review by EricSSartorius — COMMENTED** [review]
I didnt boot this locally but the code looks like it makes sense. The only thing is though that it doesn't look like these have been made to be dashboard specific. I will approve if you can confirm that it only affects the dashboard footer code on mobile and not the main site footer code since that is already fine

File: ui/src/styles/_cards.scss [hunk 2]
```diff
@@ -66,6 +66,27 @@
     border: var(--line);
   }
 
+  // dashboard edit menu cards and style
+  &--update {
```
Comment (EricSSartorius) [inline ui/src/styles/_cards.scss]: if this is specific to dashboard then maybe it makes sense to name it card--dashboard and/or add some logic where it can only be accessed within a dashboard layout (and make sure all pages in a dashboard layout have the same dashboard layout class)

Justy trying to stay away from specificity with global styles as much as possible

File: ui/src/dashboard/menus/MenuForm.tsx [hunk 3]
```diff
@@ -46,15 +46,21 @@ export const MenuForm = ({ onSubmit, menu = null }) => {
...
+              <div>
+                <Form>
+                  <Field isFormik name="title" label="Title" />
+                  <Field isFormik as="textarea" label="Description" name="description" />
+                  <Field isFormik as="textarea" label="Disclaimer" name="disclaimer" />
+                  <Button type="submit" disabled={isSubmitting}>
+                    {menu ? 'Update' : 'Create'} Menu
+                  </Button>
+                </Form>
+              </div>
```
Comment (EricSSartorius) [inline ui/src/dashboard/menus/MenuForm.tsx]: This and the card css code should be in a separate PR than the footer stuff. If for some reason one of these 2 things needs rolledback for example, we couldnt ever cherry pick a specific issue, we would have to remove both features. Just a good habbit to remember and isolate PRs and not double dip when possible :)

Comment (RB-Dev94) [reply ui/src/dashboard/menus/MenuForm.tsx]: oh snap i didnt realize i left this in that branch. 1000 apologies

**Review by EricSSartorius — COMMENTED** [review]
@ManticoreDes Is this still valid and I accidentally missed doing a review for it? Or is it outdated and can be closed?
